sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.Publish.targets

```diff
@@ -21,4 +21,28 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.Azure.Functions.Worker.Sdk.Publish.$(PublishProtocol).targets"  
           Condition="Exists('$(MSBuildThisFileDirectory)Microsoft.Azure.Functions.Worker.Sdk.Publish.$(PublishProtocol).targets')" />
 
+    <Target
+          Name="AssignFunctionsBaseImage"
+          BeforeTargets="ComputeContainerBaseImage"
+          DependsOnTargets="_FunctionsPreBuild" >
+        <PropertyGroup>
+            <_FunctionsRuntimeMajorVersion>$(_AzureFunctionsVersionStandardized.Split('-')[0].TrimStart('vV'))</_FunctionsRuntimeMajorVersion>
```

Comment:
This line can drop the `.Split('-')[0]` once https://github.com/Azure/azure-functions-dotnet-worker/pull/2676 is ported into the `feature/2.x` branch. The change would not be required, but it would clean things up to do so.

Reply:
This is addressed in the latest iteration.